### ISSUE TITLE
Inconsistency between copper wire and raw copper wire

### ISSUE THREAD

**Issue by osuphobia** [CleverRaven/Cataclysm-DDA#74278]
### Describe the bug

https://cdda-guide.nornagon.net/item/copper
https://cdda-guide.nornagon.net/item/cable
https://cdda-guide.nornagon.net/item/copper_wire

Copper wire can be made with raw copper wire, but uncrafting copper wire do not return raw copper wire, but copper instead.
That's not reasonable, because the weight of one unit of raw copper wire is 4747 mg, while the weight of one unit of copper is 5 g.

### Attach save file

N/A

### Steps to reproduce

See hhg item data above.

### Expected behavior

Change the disassembly result to raw copper wire.

### Versions and configuration

f608d5b853e0d9d8492e2bc64166d60cc5d21007

### Additional context

#73410 
Seems that item disassembly is a zone really complicated.
Maybe more audit is needed, will check the detail of it later.
If nobody's gonna deal with copper wire disassembly, I will make a PR myself.

**Comment by osuphobia** [comment]
Also the time needed should be changed. It's too long (2 minute) for just 5 cm of copper wire, I can peel it to raw wire in seconds IRL.

**Comment by TheMurderUnicorn** [comment]
I agree that copper wire should strip to raw copper wire and now just copper.  However, it would be nice to maintain a way for players to convert wire into the raw form of copper.  Yes, copper is slightly heavier than the raw copper wire.  It's my opinion this can be ignored and kept as a one to one conversion, but I imagine that's not acceptable to some people.  I will change the deconstruction of copper wire into raw copper wire.  We don't have anything set up for wire stripping, so it will remain with a cut requirement, and since you have to manually cut the wire to strip it, I will reduce it to say, idk, 8 second instead of 2 minutes per deconstruction.

Anyway I'll make a PR, the TLDR is:
- Copper wire converts to raw copper wire, 8 seconds each wire, requires cutting
- Raw wire converts to copper, 1 s each (raw wire is effectively raw copper, and their weight difference is negligible)
- And just for the sake of explanation there's no way to cheese this to get extra copper.  IE: making wire from scratch costs 200 copper effectively and you get 182 back if you were to then disassemble it all.

Submitting the PR shortly and you can provide feedback there or whatever, idk

**Comment by osuphobia** [comment]
It may sounds like nitpicking, but I need to point out that convert raw copper wire to "copper" by uncrafting is not a good idea.
They are not just differ in weight, but also in shape. While no clear description can be found, the in-game item copper should be some small plate or fragment, so what you want to do is to smash copper wire into cube, with your bare hands, in one second.
I think a better solution is to add copper_wire to some recipes that use copper.
For example, you can make chunk of copper with 50 units of copper, then 53 units of raw copper wire should also be acceptable.

**Comment by TheMurderUnicorn** [comment]
I'll be straight I'm not messing with that.  It's scrap bits of copper, I would regard small pieces of wire as scrap bits of copper and interchangeable for the majority of circumstances.  And I don't feel that adding a different copper item and adding it to 200 recipes is worth my time, personally, but that's up to whoever if they want to do that.
